### PR TITLE
Various field fixes, clean ups, and prep for "deploy"

### PR DESCRIPTION
**Pull request by abalmos**
This is a collection of commits (should probably be several PRS) that represents all of the fixes we needed in the field, various long to do clean ups, and preparing containers.

TODO:
- [x] Build containers with GitHub actions (for arm / x86_64)
<strike>- [ ] Fixup gps logger~~~
  - [ ] Create database if not present
  - [ ] Insert all GPS data
  - [ ] User management / permissions ?</strike>

Let's deal with the DB management in another line of work

### CODE REVIEW

**Comment by abalmos** [comment]
@aaron97neu @facastiblancor I suggest we deploy JVK's IB with this branch (and oats-fleet PR) and merge ASAP (just ignore the gps2tsdb stuff for now) and then start adding services as a PR.

File: ansible/avena/roles/can/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            group: root
+            mode: "0700"
+
+        - name: ensure can-watchdog.service is enabled
+          become: yes
+          systemd:
+            name: can-watchdog
+            daemon_reload: yes
+            enabled: "{{ oats_can_watchdog }}"
+            state: "{{ 'stopped' if not oats_can_watchdog | default('started') }}"
```
Comment (aaron97neu) [inline ansible/avena/roles/can/tasks/main.yml:68]: This still needs to be fixed. For the jvk deployment I used 
```
state: "{{ 'stopped' if not oats_can_watchdog else 'started' }}"
```
However that does not work if `oats_can_watchdog` is undefined and reads awkwardly. Maybe `'started' if oats_can_watchdog is 'yes' else 'stopped'`? Have not tested it to ensure it handles undefineds properly

```suggestion
            state: "{{ 'started' if oats_can_watchdog is 'yes' else 'stopped' }}"
```

Relevant documentation on inline ifs in jinja2: https://jinja.palletsprojects.com/en/2.11.x/templates/#if-expression

**Comment by aaron97neu** [comment]
@abalmos Let's get that one issue I mentioned above solved and merge this into master. Would like to make new branches off changes from this PR and it would be cleaner with them in master

**Comment by abalmos** [comment]
@aaron97neu Does that change work? I was hoping to look at this stuff this week, but it's looking like it will be pushed to next. Happy to merge with that suggested change if you can confirm things are good.

**Comment by aaron97neu** [comment]
@abalmos My suggested changes now work, had to make a few changes. 
`oats_can_watchdog: 'yes'` - can watchdog enabled and started
`oats_can_watchdog: 'abc'` - can watchdog not enabled or started
`oats_can_watchdog: 'no'` - can watchdog not enabled or started
` ` - can watchdog not enabled or started

**Pull request merged**